include/maidsafe/crux/detail/sequence_number.hpp

```diff
@@ -77,8 +78,8 @@ sequence_number<NumericType, Max>::sequence_number(const sequence_number& other)
 }
 
 template<typename NumericType, NumericType Max>
-sequence_number<NumericType, Max>::sequence_number(NumericType n)
-    : n(n)
+sequence_number<NumericType, Max>::sequence_number(NumericType other_n)
+    : n(other_n)
```

Comment:
Yes Bjorn and Peter, these code changes are not related to the warnings.  Mere attempts to get GCC to be silent without silencing it.  To no avail.